en/advanced/parameter_reference.md

```diff
@@ -1365,6 +1365,20 @@ Increasing it makes the multi-rotor wind estimates adjust more slowly</p>   </td
  <td style="vertical-align: top;">0.1 </td>
  <td style="vertical-align: top;">rad/sec</td>
 </tr>
+<tr>
```

Comment:
Remove changes to this file please. This will be auto generated from source when https://github.com/PX4/Firmware/pull/11400 is merged.